src/Collection/ExtractTrait.php

```diff
@@ -29,27 +29,27 @@ trait ExtractTrait
      * Returns a callable that can be used to extract a property or column from
      * an array or object based on a dot separated path.
      *
-     * @param string|callable $callback A dot separated path of column to follow
+     * @param string|callable $path A dot separated path of column to follow
      * so that the final one can be returned or a callable that will take care
      * of doing that.
      * @return callable
      */
-    protected function _propertyExtractor($callback): callable
+    protected function _propertyExtractor($path): callable
     {
-        if (!is_string($callback)) {
-            return $callback;
+        if (!is_string($path)) {
+            return $path;
         }
 
-        $path = explode('.', $callback);
+        $parts = explode('.', $path);
```

Comment:
This is renamed to $parts to separate it from the original $path. The parts are an array of path sections.